app/uk.gov.hmrc.helptosaveproxy/connectors/NSIConnector.scala

```diff
@@ -85,15 +86,22 @@ class NSIConnectorImpl @Inject() (auditConnector:    AuditConnector,
       .map[Either[SubmissionFailure, SubmissionSuccess]] { response ⇒
         val time = timeContext.stop()
 
+        val acNumber = response.parseJSON[AccountNumber]()
```

Comment:
you can place this inside of `case Status.CREATED ⇒`  block otherwise incase of failures `parseJSON  will blowup

Reply:
ok thanks